src/store/actions/add_todo_in_list.js

```diff
@@ -0,0 +1,13 @@
+import {
+    IS_VISIBLE_ADD_TO_DO_IN_LIST_SAGA,
+    SET_ID_FOR_ADDING_TO_DO_SAGA
+  } from "../types";
+  
+export const toggleAddToDoInList=(isVisible)=>{
+    return{type:IS_VISIBLE_ADD_TO_DO_IN_LIST_SAGA,payload:isVisible}
+}
+
+export const setIdForAddingToDo=(id)=>
+{
+    return {type:SET_ID_FOR_ADDING_TO_DO_SAGA,payload:id}
+}
```

Comment:
Fix eof in all of the files. Go through this Pr I will not point out each make sure you fix everything.

Reply:
done